main/acle.md

```diff
@@ -5998,6 +5987,19 @@ vectors using [`svundef`](https://developer.arm.com/architectures/instruction-se
 [`svundef3`](https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=svundef3) and
 [`svundef4`](https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=svundef4).
 
+ACLE provides two sets of intrinsics for converting between vector types and vector tuple types:
```

Comment:
Sorry for the pedantic comment, but reading this fresh, it seemed like it was describing conversions from vector types to vector tuple types and vice versa.  How about adding “between” before “vector tuple types” too?  Alternatively, we could just say “between SVE types”.